dotcom-rendering/src/layouts/FootballDataPageLayout.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import type { StoryObj } from '@storybook/react';
+import { footballData } from '../../fixtures/generated/football-live';
+import { initialDays, regions } from '../../fixtures/manual/footballData';
+import { extractNAV } from '../model/extract-nav';
+import { FootballDataPageLayout } from './FootballDataPageLayout';
+
+const meta = {
+	title: 'Components/Football Data Page Layout',
+	component: FootballDataPageLayout,
+};
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Results = {
+	args: {
+		footballData: {
+			matchesList: initialDays,
+			regions,
+			kind: 'Result',
+			guardianBaseURL: 'https://www.theguardian.com',
+			editionId: 'UK',
+			config: footballData.config,
+			nav: extractNAV(footballData.nav),
+			pageFooter: footballData.pageFooter,
+			contributionsServiceUrl: 'https://contributions.guardianapis.com',
+			isAdFreeUser: false,
+		},
+	},
+} satisfies Story;
```

Comment:
I think it might be better if we use the generated fixture which is showing the real data for the live page?
For result and fixture pages, we don't yet have the data, but we can either generate them now in this PR or when we work for those pages

Reply:
After chat we decided not to go with this data as it's causing some dependancy issues in the story